_posts/2017/2017-03-22-microxchg-2017-microservices-conference.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "microXchg 2017 - Microservices conference in Berlin"
+date: "2017-02-22 12:02:22"
+image: blog-header/microxchg-2017.jpg
```

Comment:
Please check for another header image. It doesn't fit as in the detail view many people are displayed without their body or heads ;-)
Something without people would be better. We could also use the standard image for the category.